example/counter.dart

```diff
@@ -0,0 +1,117 @@
+/// Copyright (C) 2021 Intel Corporation
```

Comment:
Why is this file here?  This looks like a copy of example.dart?

Reply:
Yes, let me change the dependent to `'import ./example.dart'` instead.